OrbitGl/App.cpp

```diff
@@ -1196,8 +1261,23 @@ void OrbitApp::UpdateProcessAndModuleList(int32_t pid) {
           }
         }
       }
+      absl::flat_hash_map<std::string, std::vector<uint64_t>> frame_track_function_hashes_map;
+      for (const FunctionInfo& func :
+           data_manager_->user_defined_capture_data().frame_track_functions()) {
+        const ModuleData* module = module_manager_->GetModuleByPath(func.loaded_module_path());
+        if (!process->IsModuleLoaded(module->file_path())) {
+          RemoveFrameTrack(func);
+        } else {
+          if (!module->is_loaded()) {
+            RemoveFrameTrack(func);
+            frame_track_function_hashes_map[module->file_path()].push_back(
+                FunctionUtils::GetHash(func));
+          }
+        }
```

Comment:
```suggestion
        } else if (!module->is_loaded()) {
            RemoveFrameTrack(func);
            frame_track_function_hashes_map[module->file_path()].push_back(
                FunctionUtils::GetHash(func));
        }
```
Also maybe it worth to add an explanatory comment for 'if' branches.

Reply:
Thanks, good suggestion. Fixed.